### PR TITLE
Add merge schedule action

### LINKED ISSUE #200: Schedule blog posts
We've had multiple blog posts come in rather quickly after one another recently. In a technical sense, that is not such an issue, but from the perspective of pacing the content this is problematic.

The risk is that we create highly irregular and volatile posting schedules (e.g.,, flooding with posts one month and then none for a while). This can reduce engagement with the content in the long run. 

Might it be an idea to think about ways to schedule these posts in the near future?

### PR DESCRIPTION
**Pull request by chartgerink**
This PR adds an action to help with scheduling pull requests. It is based on the [`merge-schedule`](https://github.com/marketplace/actions/merge-schedule) action. It only merges PRs at the scheduled time if all the conditions are fulfilled:

- Checks pass
- Rebase merge possible
- Original PR comment includes a `/schedule <datetime>` command

This will helps us schedule blog posts, and prevent flooding the blog with posts at one time. We do not yet have a schedule itself, so this will be something a future community manager can set up.

Fixes #200.

### CODE REVIEW

File: .github/workflows/scheduler.yml [hunk 1]
```diff
@@ -0,0 +1,29 @@
+name: Merge Schedule
+
+on:
+  pull_request:
+    types:
+      - opened
+      - edited
+      - synchronize
+  schedule:
+    # https://crontab.guru/every-hour
```
Comment (Bisaloo) [inline .github/workflows/scheduler.yml:10]:
```suggestion
```

Outdated comment

File: README.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+```
+quarto render
+# individual pages
+quarto render <file>
+```
+
+## Scheduling pull requests
+
+Given that we sometimes want to merge pull requests at specific times, especially for blog posts, there is the option to schedule a merge attempt. To do so, add an ISO8601 compliant datetime command as such to the first post in the PR: `/schedule <datetime>` (e.g., `/schedule 2026-01-01` for January 1st, 2026).
```
Comment (Bisaloo) [inline README.md:17]:
```suggestion
Given that we sometimes want to merge pull requests at specific times, especially for blog posts, there is the option to schedule a merge attempt. To do so, add an ISO8601 compliant datetime command as such at the end of the first post in the PR: `/schedule <datetime>` (e.g., `/schedule 2026-01-01` for January 1st, 2026).
```

Not sure. This is based on my understanding of the `merge-schedule` docs

**Review by Bisaloo — COMMENTED** [review]
I suspect the GHA job will require explicit permissions to work given the recent changes in GHA security.

Docs: https://docs.github.com/en/actions/using-jobs/assigning-permissions-to-jobs

**Comment by chartgerink** [comment]
Thanks @Bisaloo - checked with the action repo and its issues, and added the recommended permissions (`write` for contents and pull-requests).

**Review by Bisaloo — APPROVED** [review]

**Pull request merged**